book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly set `attr_accessor` for rentals also as you would need to both access and modify it.

Reply:
Thank you will do.